src/GogoKit/Models/Response/SellerListing.cs

```diff
@@ -78,6 +78,13 @@ public class SellerListing : EmbeddedSellerListing
         [Embedded("listing_notes")]
         public IList<ListingNote> ListingNotes { get; set; }
 
+        /// <summary>
+        /// The model for the transaction that this listing was created from
+        /// </summary>
+        [DataMember(Name = "relist_transaction")]
+        [Embedded("relist_transaction")]
+        public virtual RelistTransaction RelistTransaction { get; set; }
```

Comment:
I don't think we should make this virtual

Reply:
removed